packages/tokamak/contracts/deploy/019-Tokamak-TONFeeVault.deploy.ts

```diff
@@ -0,0 +1,22 @@
+/* Imports: External */
```

Comment:
I think if we execute `hardhat deploy --network mainnet`, this file will execute. Do you know how can we avoid it? Or, can you share other details?

Reply:
Great! only for liveNetworks. Thank you very much!